internal/fleet/package_resource.go

```diff
@@ -0,0 +1,120 @@
+package fleet
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/elastic/terraform-provider-elasticstack/internal/clients/fleet"
+	"github.com/elastic/terraform-provider-elasticstack/internal/utils"
+)
+
+func getPackageID(name, version string) string {
+	hash, _ := utils.StringToHash(name + version)
+
+	return *hash
+}
+
+func ResourcePackage() *schema.Resource {
+	packageSchema := map[string]*schema.Schema{
+		"name": {
+			Description: "The package name.",
+			Type:        schema.TypeString,
+			Required:    true,
+		},
+		"version": {
+			Description: "The package version.",
```

Comment:
Doesn't look like it, unfortunately. I tried putting in `latest` and Kibana kicked back a 500 (also did this just hitting the API directly through Postman). I checked the logs and it just said `500 server error` 🤷 

```
[2023-10-31T13:13:37.007+00:00][ERROR][http] 500 Server Error
```

I believe it wants a semver, since if you give it something that looks more like semver, but isn't, then it produces a semver validation error. If I had to guess, Kibana is throwing an exception trying to parse `latest` as a semver, but not surfacing the error.